FhirToDataLake/azurepipelines/ci.yml

```diff
@@ -110,7 +110,7 @@ steps:
     targetType: inline
     workingDirectory: FhirToDataLake
     script: |
-      docker build -t $(imageName):$(version) -f ./src/Microsoft.Health.Fhir.Synapse.Tool/Dockerfile .
+      docker build -t $(imageName):$(version) -f ./src/Microsoft.Health.Fhir.Synapse.Tool/Dockerfile . --build-arg Version=$(version)
```

Comment:
👍

Reply:
Maybe we can use another parameter like `binaryversion` or `assemblyversion` to avoid ambiguity